src/net/RTCP/RTCPCompoundPacket.cs

```diff
@@ -100,9 +102,20 @@ public RTCPCompoundPacket(byte[] packet)
                             break;
                         case (byte)RTCPReportTypesEnum.RTPFB:
                             // TODO: Interpret Generic RTP feedback reports.
-                            Feedback = new RTCPFeedback(buffer);
-                            int rtpfbFeedbackLength = (Feedback != null) ? Feedback.GetBytes().Length : Int32.MaxValue;
-                            offset += rtpfbFeedbackLength;
+                            var typ = new RTCPHeader(buffer).FeedbackMessageType; ;

```

Comment:
`buffer` is being parsed and a `RTCPHeader` instance is being created just for using the `FeedbackMessageType`.

Consider parsing the buffer just to collect the `FeedbackMessageType`.